onward/app/feed/MostPopularAgent.scala

```diff
@@ -59,8 +61,17 @@ class GeoMostPopularAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi
         item: JsValue <- ophanResults.asOpt[JsArray].map(_.value).getOrElse(Nil)
         url <- (item \ "url").asOpt[String]
       } yield {
-        contentApiClient.getResponse(contentApiClient.item(urlToContentPath(url), Edition.defaultEdition))
-          .map(_.content.map(RelatedContentItem(_)))
+        contentApiClient
+          .getResponse(contentApiClient
+            .item(urlToContentPath(url), edition)
+            .showTags("paid-content")
+            .showSection(true)
+            .showFields("isAppropriateForSponsorship"))
+          .map(_.content
+            .filterNot { content =>
+              val branding: Option[Branding] = BrandingFinder.findBranding(content, countryCode)
+              branding.exists(_.isPaid)}
```

Comment:
I think actually we will need it as we will also filter out sponsorship.
` branding.exists(_.isPaid) &&  branding.exists(_.isSponsorship)`

I have to clear this up yet though.